### DISCUSSION TITLE
Topic Creation with Additional Config

### DISCUSSION THREAD

**Discussion by n-i-c-o-l-a-s** [tchiotludo/akhq#859] [Ideas]
Hello. We are working for one of the largest Bank in the world and we love AKHQ to administrate our Kafka Cluster.

In the topic's creation page (/topic/create in AKHQ 0.18.0), it doesn't seems possible to define additional configurations.

One of our goal would be to define the following configs :
- confluent.value.schema.validation=true / false
- confluent.key.schema.validation=true / false

Unix command example: _kafka-topics --create --bootstrap-server localhost:9092 --replication-factor 1 --partitions 1 --topic my-topic-sv \
--**config confluent.value.schema.validation=true**_

Is it something planned soon ? 

Thank you.

**Comment by tchiotludo** [comment]
Hi, 

Thanks for the report ! 
If your bank can do [this](https://github.com/tchiotludo/akhq/discussions/161), I'll really appreciate  :smile: 

I create the issues https://github.com/tchiotludo/akhq/issues/860 in order to keep track on these. This seems to be a really simple feature, does your bank have some dev to make a PR ?

**Comment by n-i-c-o-l-a-s** [comment]
Hello. Local developers can make a PR. We will see this in a few weeks.

**Comment by n-i-c-o-l-a-s** [comment]
Is it possible to customize default topic configuration when we create one ? 

For example, the number of partitions and replication factor

**Comment by tchiotludo** [comment]
yes, look at [this](https://akhq.io/docs/configuration/akhq.html#topic-creation-default-values)

**Comment by n-i-c-o-l-a-s** [comment]
Thanks a lot.

We run the JAR with : _java -Dmicronaut.config.files=application-dev.yml -jar akhq.jar_

And our application-dev.yml contains things like that:

![image](https://user-images.githubusercontent.com/44503667/150542339-bf2ae335-6519-4e4e-871d-76e8722e8704.png)

It seems it doesn't override your application.yml located at the root of _akhq.jar_ 

Any idea ?

**Reply by tchiotludo** [reply]
try absolute path maybe ? 
https://github.com/tchiotludo/akhq/blob/dev/helm/akhq/templates/deployment.yaml#L71-L72

**Comment by n-i-c-o-l-a-s** [comment]
For information, the app is not running using Docker.

**Reply by tchiotludo** [reply]
i understand ;)
